x-pack/packages/kbn-ai-assistant/src/hooks/use_conversation.ts

```diff
@@ -62,21 +60,17 @@ export function useConversation({
   connectorId,
   onConversationUpdate,
 }: UseConversationProps): UseConversationResult {
-  const service = useObservabilityAIAssistantAppService();
+  const service = useAIAssistantAppService();
   const { scope } = service;
 
   const {
     services: {
       notifications,
-      plugins: {
-        start: {
-          observabilityAIAssistant: { useChat },
-        },
-      },
+      observabilityAIAssistant: { useChat },
     },
   } = useKibana();
 
-  const initialConversationId = useOnce(initialConversationIdFromProps);
+  const initialConversationId = initialConversationIdFromProps;
```

Comment:
why was `useOnce` removed?

Reply:
Reverted in https://github.com/elastic/kibana/pull/194552/commits/911c57e4768d24833c236d942d6c860f4fed41f9